modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2AnalyticsCloudSetup.macro

```diff
@@ -0,0 +1,83 @@
+definition {
+
+	macro activeStatus {
+		ApplicationsMenu.gotoPortlet(
+			category = "Object",
+			panel = "Control Panel",
+			portlet = "Account Subscription Groups");
+
+		CustomerPortal.viewObjectInfo(index = "3");
+
+		CP2AnalyticsCloudSetup.typeSetupAnalyticsCloudInfoFields(
+			label = "activationStatus",
+			value = "Active");
+
+		Click(locator1 = "Button#SAVE");
+
+		Alert.viewSuccessMessage();
+	}
+
+	macro clearAndDeleteCloudEntries {
+		CustomerPortal.clearActivationStatus(status = "${status}");
+
+		ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+
+		CP2DXPCloudSetup.deleteEntryCloudEnvironments(
+			entry = "ERC-003",
+			portlet = "Analytics Cloud Workspaces");
```

Comment:
@luanakarla is there a specific reason for us to delete only "ERC-003" entry? I am asking that because this macro will only work for ERC-003. Do you think it would be better if this macro could work for any entry and not only ERC-003? Is there a chance to have this macro used with other entry?

Reply:
Oh yes, I made it more generic.